[raylib] LNK4044 warning because of GCC flags in .vcxproj

Building a project using raylib with CMake and MSVC gives an LNK4044 warning:

[build] LINK : warning LNK4044: unrecognized option '/LC:/vcpkg/installed/x64-windows/debug/lib -lraylib'; ignored [D:\VSCode\C\gametest\build\gametest.vcxproj]

These are GCC flags and shouldn't make it into the .vcxproj file.

My CMakeLists.txt contains:

find_package(raylib CONFIG REQUIRED)
target_link_libraries(main PRIVATE raylib)

as vcpkg specifies when raylib is installed. There is no relevant source code because I just compiled a hello world program as a sanity check. I added libpng in the same way, which didn't generate the linker warning.

--- a/ports/raylib/fix-link-path.patch
+++ b/ports/raylib/fix-link-path.patch
@@ -1,0 +1,13 @@
+diff --git a/cmake/raylib-config.cmake b/cmake/raylib-config.cmake
+index 700965c..4815cd6 100644
+--- a/cmake/raylib-config.cmake
++++ b/cmake/raylib-config.cmake
+@@ -65,7 +65,7 @@ if (NOT TARGET raylib)
+       IMPORTED_LOCATION             "${raylib_LIBRARIES}"
+       IMPORTED_IMPLIB               "${raylib_LIBRARIES}"
+       INTERFACE_INCLUDE_DIRECTORIES "${raylib_INCLUDE_DIRS}"
+-      INTERFACE_LINK_LIBRARIES      "${raylib_LDFLAGS}"
++      INTERFACE_LINK_LIBRARIES      "${raylib_LIBRARIES}"
+       INTERFACE_COMPILE_OPTIONS     "${raylib_DEFINITIONS}"
+     )
+ 

--- a/ports/raylib/portfile.cmake
+++ b/ports/raylib/portfile.cmake
@@ -18,6 +18,7 @@ vcpkg_from_github(
     HEAD_REF master
     PATCHES
         android.diff
+        fix-link-path.patch
 )
 file(GLOB vendored_headers RELATIVE "${SOURCE_PATH}/src/external"
     "${SOURCE_PATH}/src/external/cgltf.h"

--- a/ports/raylib/vcpkg.json
+++ b/ports/raylib/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "raylib",
   "version": "5.0",
-  "port-version": 1,
+  "port-version": 2,
   "description": "A simple and easy-to-use library to enjoy videogames programming",
   "homepage": "https://github.com/raysan5/raylib",
   "license": "Zlib",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -7774,7 +7774,7 @@
     },
     "raylib": {
       "baseline": "5.0",
-      "port-version": 1
+      "port-version": 2
     },
     "rbdl": {
       "baseline": "3.3.0",

--- a/versions/r-/raylib.json
+++ b/versions/r-/raylib.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "e16299f8e3d6b40674ce53b106b764bd7459ad63",
+      "version": "5.0",
+      "port-version": 2
+    },
+    {
       "git-tree": "6ef397a5c5afb217736842e45d9610d084a1fb2d",
       "version": "5.0",
       "port-version": 1

